tests/cpp/build_module_test.cc

```diff
@@ -0,0 +1,42 @@
+#include <dmlc/logging.h>
+#include <gtest/gtest.h>
+#include <tvm/tvm.h>
+#include <tvm/operation.h>
+#include <tvm/build_module.h>
+
+TEST(BuildModule, Basic) {
+  using namespace tvm;
+  auto n = Variable::make(Int(32), "n");
```

Comment:
Use Var("n")

Reply:
This can only use Var("n") if the constructor of Halide::VarExpr is marked EXPORT - is this OK? I was originally using Variable::make so as not to require exposing the implementation detail from the Halide namespace

Reply:
I see,  I think we can create a function var, which constructs Var and export that function, so it is consistent with python